bootstrap.sh

```diff
@@ -15,6 +15,8 @@ kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
 kubectl apply -f .infrastructure/app/deployment.yml
 
+kubectl apply -f .infrastructure/security/rbac.yml
```

Comment:
The script applies the RBAC configuration, which is crucial for setting up permissions. Ensure that the `rbac.yml` file is correctly configured to avoid permission issues.